src/components/markdown/ProblemsList/ProblemStatusCheckbox.tsx

```diff
@@ -9,124 +9,76 @@ import UserDataContext from '../../../context/UserDataContext/UserDataContext';
 import Tippy from '@tippyjs/react';
 import 'tippy.js/themes/light.css';
 import styled from 'styled-components';
-
+import feedbackModal from '../../feedbackModal';
 import MarkdownLayoutContext from '../../../context/MarkdownLayoutContext';
 import ConfettiContext from '../../../context/ConfettiContext';
+import Transition from '../../Transition';
+import { logError } from 'gatsby/dist/state-machines/develop/actions';
+import * as firebase from 'firebase';
+import FirebaseContext from '../../../context/FirebaseContext';
 
 const StyledTippy = styled(Tippy)`
   .tippy-content {
     padding: 0;
   }
 `;
-
-const ProgressDropdown = ({ onProgressSelected, currentProgress }) => {
+// testing merge
+const ProgressDropdown = ({
+  onProgressSelected,
+  currentProgress,
+  openModal,
+}) => {
   const [activeProgress, setActiveProgress] = useState<ProblemProgress>();
 
-  const icon = (status: ProblemProgress, equal: Boolean) => {
```

Comment:
It looks like this code got lost during the merge? Ben added icons to the problem dropdown

Reply:
hmm can I manually place it back?